modules/ROOT/pages/release-notes/release-notes56.adoc

```diff
@@ -160,9 +163,9 @@ The {productname} 5.6 release includes an accompanying release of the *PowerPast
 *PowerPaste* 5.4.0 also provides the following bug fixes:
 
 * Fixed the `Cut` menu item not working in the latest version of Firefox.
-* Fixed two Cross-Site Scripting (XSS) vulnerability issues. For more information, see: <<security-fixes,Security fixes>>.
+* Fixed two Cross-Site Scripting (XSS) vulnerability issues. For more information, see: xref:security-fixes[Security fixes].
```

Comment:
```suggestion
* Fixed two Cross-Site Scripting (XSS) vulnerability issues. For more information, see: xref:securityfixes[Security fixes].
```
This xref doesn't resolve as the id added below doesn't have a hypen.

Reply:
done